modules/dnn/misc/python/test/test_dnn.py

```diff
@@ -474,6 +470,31 @@ def test_input_3d(self):
 
             normAssert(self, real_output, gold_output, "", getDefaultThreshold(target))
 
+    def test_set_param_3d(self):
```

Comment:
@asmorkalov A new test for `setParam` is added anyway. Tested without `CV_ND` and it fails.